src/app/dashboard/flight-table/flight-table.component.html

```diff
@@ -106,5 +139,5 @@
     [pageIndex]="routedParams?.page - 1 || 0"
     [pageSizeOptions]="pageSizeOptions"
     showFirstLastButtons
-  ></mat-paginator>
```

Comment:
Whoops, this extra bit got left in there

Reply:
Fixed